py-polars/pyproject.toml

```diff
@@ -184,6 +184,8 @@ filterwarnings = [
   "error",
   # ...except where it prevents test debugging in an IPython console
   "ignore:.*unrecognized arguments.*PyDevIPCompleter:DeprecationWarning",
+  # or for this deprecation warning
+  "ignore:`Series.time_unit` is deprecated and will be removed in a future version, please use `Series.dtype.time_unit` instead:DeprecationWarning",
```

Comment:
Without this ignore, 42 tests fails. I'm not sure whether we should put `warnings` catch on all of them or just ignore them from here is fine ?

Reply:
which ones fail? I suspect it's cases when internally polars is using `.time_unit` - in which case, you might be able to update that to `.dtype.time_unit` and avoid the warning

Reply:
yeah I think type: ignore is fine, as not all dtypes have `time_unit` and `time_zone`